x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.tsx

```diff
@@ -151,13 +162,24 @@ export const HostList = () => {
         name: i18n.translate('xpack.endpoint.host.list.os', {
           defaultMessage: 'Operating System',
         }),
+        truncateText: true,
       },
       {
         field: 'metadata.host.ip',
         name: i18n.translate('xpack.endpoint.host.list.ip', {
           defaultMessage: 'IP Address',
         }),
-        truncateText: true,
+        render: (ip: string[]) => {
+          return (
+            <EuiToolTip content={ip.toString().replace(',', ', ')} anchorClassName="eui-fullWidth">
```

Comment:
Suggestion: do the `ip.toString().replace(',', ', ')` once and store value in a variable - then use variable in the two places below.